MultiSigWalletOverride/migrations/2_deploy_contracts.js

```diff
@@ -4,25 +4,33 @@ const MultisigWalletFactory = artifacts.require('MultiSigWalletWithDailyLimitFac
 const fs = require('fs');
 const path = require('path');
 
-module.exports = deployer => {
+module.exports = async (deployer, [_deployerAddress]) => {
   const args = process.argv.slice();
-  if (process.env.DEPLOY_FACTORY){
-    deployer.deploy(MultisigWalletFactory);
-    console.log("Factory with Daily Limit deployed");
-  } else if (args.length < 5) {
-    console.error("Multisig with daily limit requires to pass owner " +
-      "list, required confirmations and daily limit");
-  } else if (args.length < 6) {
-    deployer.deploy(MultisigWalletWithoutDailyLimit, args[3].split(","), args[4]).then(function() {
+  let accountsIndex = undefined;
+  let confirmationsIndex = undefined;
+  for (let j = 0; j < process.argv.length; j++) {
+    
+    if (process.argv[j] == '--accounts') {
+      accountsIndex = j;
+    }
+    if (process.argv[j] == '--confirmations') {
+      confirmationsIndex = j;
+    }
+  }
+  console.log(`Accounts: ${args[accountsIndex + 1].split(",")}`);
```

Comment:
minor: can we put `args[accountsIndex + 1].split(",")` into a variable as this is used afterward again to be more readable? same as the confirmation one.